identifiers/country-in/vidhan_sabha.csv

```diff
@@ -1,18 +1,379 @@
 id,name

```

Comment:
It looks like consituencies for Madhya Pradesh and Jammu and Kashmir are missing?

Reply:
Good call, corrected typo for Madhya Pradesh.

Jammu & Kashmir was considered a state, but since the end of 2019, the state was dissolved and it is now split into two Union Territories - J&K and Ladakh. They are currently under a delimitation exercise to redraw the constituencies.

Reply:
Thanks for fixing the typo and the clarification.